pkg/internal/checkup/checkup.go

```diff
@@ -198,8 +198,8 @@ func (c *Checkup) waitForVMIDeletion(ctx context.Context) error {
 func newRealtimeVMI(checkupConfig config.Config) *kvcorev1.VirtualMachineInstance {
 	const (
 		CPUSocketsCount   = 1
-		CPUCoresCount     = 3
-		CPUTreadsCount    = 1
+		CPUCoresCount     = 2
+		CPUTreadsCount    = 2
```

Comment:
Currently we have a guest with SMT disabled (CPUTreadsCount    = 1).
I'm not sure we need to change that.
@matosatti WDYT?

Reply:
my resons for adding this: 
- This is how we did it on the DPDK repo
- Our stakeholders wanted the checkup to run on a SMT enabled environment, so I guess this is what they mean..

Reply:
In https://access.redhat.com/solutions/7007632 (which serves as our reference), SMT is disabled in the guest.

Reply:
OK then let's wait for @matosatti before we proceed

Reply:
talked offline - changed to `CPUTreadsCount =1`. 
But the question remains open - it will be addressed on a separate PR if needed.

Reply:
It should not matter whether SMT is disabled or not in the guest vCPUs.